### ISSUE TITLE
What's wrong with this definitions spec?

### ISSUE THREAD

**Issue by davisford** [swagger-api/swagger-editor#799]
```yaml
definitions:
  Model:
    type: object
    description: Common properties to most model object
    properties:
      id: {type: number, format: int, readOnly: true}
      createdAt: {type: string, format: date-time, readOnly: true}
      updatedAt: {type: string, format: date-time, readOnly: true}
  Reward:
    type: object
    properties:
      allOf: {$ref: '#/definitions/Model'}
      rewardTypeId: {type: number, format: int, readOnly: true}
  Pager:
    type: object
    properties:
      limit: {description: the limit that was passed to the server, type: number,
        format: int, readOnly: true}
      offset: {description: the offset that was passed to the server, type: number,
        format: int, readOnly: true}
      total: {description: the total number of items in the database, type: number,
        format: int, readOnly: true}
  RewardPager:
    type: object
    properties:
      allOf: {$ref: '#/definitions/Pager'}
      items:
        description: array of reward objects
        type: array
        items: {$ref: '#/definitions/Reward'}
```

Why does this result in an undefined array type when the spec is parsed?

![screen shot 2016-01-29 at 12 44 32 pm](https://cloud.githubusercontent.com/assets/186677/12683230/1f97908e-c686-11e5-850a-6a173b7c65b2.png)

**Comment by webron** [comment]
Not sure why it doesn't render properly, but I have a feeling you're not modeling what you expect... Does your `RewardPager` really have a property called `allOf`? Not impossible, just checking.

**Comment by saharj** [comment]
This issue has been fixed in this pull request: #787

**Comment by davisford** [comment]
Cool, do you happen to know when the change will hit editor.swagger.io?
